frontend/components/vault/VaultsText.vue

```diff
@@ -0,0 +1,125 @@
+                {{ vaultTransactions.length }} vault<span v-if="vaultTransactions.length !== 1">s</span>,
+                {{ vaultsReadyToBeLiquidated }} of which {{ vaultsReadyToBeLiquidated === 1 ? 'is' : 'are' }} ready to
+                be liquidated. {{ liquidatedVaults }} vault<span v-if="liquidatedVaults !== 1">s</span>
+                {{ liquidatedVaults === 1 ? 'has' : 'have' }} already been liquidated.
+            </TextBlock>
+            <TextBlock v-else title="No at risk vaults" class="TextBlock">
+                There are currently not vaults at risk. You can check <nuxt-link to="/collateral">here</nuxt-link> to
+                see if there are any ongoing collateral auctions.
+            </TextBlock>
+        </template>
+        <div
+            class="w-full self-center"
+            :class="{ 'max-w-4xl': isExplanationsShown, 'md:px-10': !isExplanationsShown }"
+        >
+            <!-- Add VaultsTable -->
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import type { VaultTransaction } from 'auctions-core/src/types';
+import { Alert } from 'ant-design-vue';
+import WhatIsMakerProtocol from '~/components/common/other/WhatIsMakerProtocol.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import Explain from '~/components/common/other/Explain.vue';
+
+export default Vue.extend({
+    components: {
+        TextBlock,
+        WhatIsMakerProtocol,
+        Alert,
+        Explain,
+    },
+    props: {
+        vaultTransactions: {
+            type: Array as PropType<VaultTransaction[]>,
+            default: () => [],
+        },
+        isVaultsLoading: {
+            type: Boolean,
+            default: false,
+        },
+        vaultsError: {
+            type: String,
+            default: null,
+        },
+        selectedVaultId: {
+            type: String,
+            default: null,
+        },
+        isExplanationsShown: {
+            type: Boolean,
+            default: true,
+        },
+        lastUpdated: {
+            type: Date,
+            default: null,
+        },
+    },
+    computed: {
+        demoURL(): string {
+            return `${process.env.FRONTEND_ORIGIN || ''}/vaults?vault=COLLATERAL_ILK:VAULT_ID`;
+        },
+        liquidatedVaults(): number {
+            let liquidatedVaults = 0;
+            this.vaultTransactions.forEach(vault => {
+                if (vault.state === 'liquidated') {
+                    liquidatedVaults++;
+                }
+            });
+            return liquidatedVaults;
+        },
+        vaultsReadyToBeLiquidated(): number {
+            let vaultsReadyToBeLiquidated = 0;
+            this.vaultTransactions.forEach(vault => {
+                if (vault.state === 'liquidatable') {
+                    vaultsReadyToBeLiquidated++;
+                }
+            });
+            return vaultsReadyToBeLiquidated;
```

Comment:
You can easily avoid create another variable using filter 
```suggestion
            return this.vaultTransactions.filter(vault => {
                return vault.state === 'liquidatable';
            }).length;
```